core/src/main/kotlin/org/evomaster/core/problem/graphql/GqlConst.kt

```diff
@@ -53,7 +53,15 @@ object GqlConst {
     const val QUERY = "query"
     const val QUERY_TYPE = "querytype"
     const val ROOT = "root"
+    const val QUERY_ROOT = "query_root"
+    const val ROOTQUERY = "rootquery"
+    const val ROOTQUERYTYPE = "rootquerytype"
+    const val DOTAQUERY = "dotaquery"
+
     const val MUTATION = "mutation"
+    const val ROOTMUTATIONTYPE = "rootmutationtype"
+    const val ROOTMUTATION = "rootmutation"
+    const val DOTAMUTATION = "dotamutation"
 
```

Comment:
please add some comments/documentation for these constants. eg, what is the difference between `QUERY_ROOT` and `ROOTQUERY`?

Reply:
did u add the documentation? or, if those are not used anymore, should be removed from the code